postgresql/src/test/scala/akka/persistence/postgresql/journal/PostgreSqlJournalSpec.scala

```diff
@@ -27,34 +27,42 @@ import scala.concurrent.duration._
 
 object PostgreSqlJournalSpec {
 
-  private val JournalPluginConfig = ConfigFactory.parseString(
-    """
-      |akka.persistence.journal.plugin = "postgresql-journal"
-      |akka.loglevel = "DEBUG"
-      |""".stripMargin)
+  private val JournalPluginConfig = ConfigFactory.parseString("""
```

Comment:
Can you align this a bit, so the actual config is not at the right side of the screen? I believe a few other configs  above could benefit from the same.